spec/requests/assign_controller_spec.rb

```diff
@@ -4,7 +4,10 @@
 require_relative "../support/assign_allowed_group"
 
 RSpec.describe DiscourseAssign::AssignController do
-  before { SiteSetting.assign_enabled = true }
+  before do
+    SiteSetting.assign_enabled = true
+    SiteSetting.enable_assign_status = true
```

Comment:
Can we move this to a context so that it is not enabled for all the existing specs, but rather only for the new specs that are specific to the status feature?